mne/preprocessing/nirs/_tddr.py

```diff
@@ -0,0 +1,143 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
```

Comment:
remove my name. Keep the name of the people who wrote the code in the file. thanks :)

Reply:
Sorry that's a copy paste job.